install_config/web_console_customization.adoc

```diff
@@ -174,6 +174,36 @@ assetConfig:
 ====
 endif::[]
 
+[[enable-tech-preview-feature]]
+=== Enabling Features in Tech Preview
+
+Sometimes features will be available in tech preview, by default these features may be hidden
+from end users.  
+
+The current list of features in tech preview is:
+
+* pipelines - Enabling this will add the Pipelines navigation item underneath the Builds menu
+
+To enable a feature:
+
+====
+----
+window.OPENSHIFT_CONSTANTS.ENABLE_TECH_PREVIEW_FEATURE.feature_name = true;
+----
+====
```

Comment:
I'm not sure what to do with this. Judging by the next step, this is a script? Do they just throw this into the command line? Is this supposed to be in an edit in a file somewhere?